src/index.html

```diff
@@ -1,13 +1,52 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <a href="#" class="card__image-link">
+        <img
+          src="images/imac.jpeg"
+          alt="iMac image"
+          class="card__image"
+          >
```

Comment:
https://github.com/mate-academy/layout_product-cards/pull/3712#discussion_r1365092370 - here the mentor indicated, along with indentation, how to format this part of the code nicely.
```suggestion
        <img
          src="images/imac.jpeg"
          alt="iMac image"
          class="card__image"
        >
```